doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -74,14 +74,14 @@ In order for the Kubernetes components to be conformant with the Reference Archi
 |Ref|Specification|Details|Requirement Trace|Reference Implementation Trace|
 |---|---|---|---|---|
 |`ra2.k8s.001`|Kubernetes Conformance|The Kubernetes distribution, product, or installer used in the implementation must be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1uF9BoDzzisHSQemXHIKegMhuythuq_GL3N1mlUUK2h0/edit#gid=0) and marked (X) as conformant for the Kubernetes version that is being used.|[req.gen.cnt.03](./chapter02.md#23-kubernetes-architecture-requirements)||
```

Comment:
If I am not mistaken, the underlying installer of BMRA is Kubespray, which is in fact listed. Worth a comment in the table?

Reply:
The reason it wasn't included was that Kubespray wasn't certified for K8s 1.19 - But I can see it got the certification about an hour before you added this comment, so I will update the table :)

Reply:
oh, I didn't notice the timing! Thank you.